package-res/resources/web/dojo/pentaho/common/overrides/dojo/_base/kernel.js

```diff
@@ -10,6 +12,7 @@ define(["../has", "./config", "require", "module"], function(has, config, requir
 
 		// create dojo, dijit, and dojox
 		// FIXME: in 2.0 remove dijit, dojox being created by dojo
+		global = (function () { return this; })(),
 		dijit = {},
```

Comment:
Using a global variable determined like this guards us against exactly that problem!